commercetools-models/src/main/java/io/sphere/sdk/orders/OrderImportDraft.java

```diff
@@ -57,7 +57,7 @@
     Address getShippingAddress();
 
     @Nullable
-    OrderShippingInfo getShippingInfo();
+    ShippingInfoImportDraft getShippingInfo();
```

Comment:
I didn't see this breaking (?) change documented in our release notes?!

Reply:
Good catch, would update the doc.